docs/content/latest/develop/ecosystem-integrations/apache-spark/java.md

```diff
@@ -37,105 +36,130 @@ isTocNested: true
 
 </ul>
 
-## Before you begin
-
-### Maven
+## Setting Up the Project with Maven
 
-To build your Java application using the YugabyteDB Spark Connector for YCQL, add the following snippet to your `pom.xml` for Scala 2.11:
+To build a Java application using the YugabyteDB Spark Connector for YCQL, add the following snippet to your `pom.xml` file for Scala 2.12:
 
 ```xml
 <dependency>
   <groupId>com.yugabyte.spark</groupId>
-  <artifactId>spark-cassandra-connector_2.11</artifactId>
-  <version>2.4-yb-3</version>
+  <artifactId>spark-cassandra-connector_2.12</artifactId>
+  <version>3.0-yb-1</version>
```

Comment:
We've update this since, let's use the most recent version instead:
```
3.0-yb-8
```

Reply:
Changed